recipes/libsass/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class LibsassConan(ConanFile):
+    name = "libsass"
+    license = "MIT"
+    homepage = "libsass.org"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "A C/C++ implementation of a Sass compiler"
+    topics = ("Sass", "LibSass", "compiler")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    build_requires = "autoconf/2.69", "libtool/2.4.6"
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def validate(self):
+        if self.settings.os not in ["Linux", "FreeBSD", "Macos"]:
+            raise ConanInvalidConfiguration("libsass supports only Linux, FreeBSD and Macos")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        tools.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self.run("autoreconf -fiv", run_environment=True)
+        self._autotools = AutoToolsBuildEnvironment(self)
+        args = []
+        args.append("--disable-tests")
+        args.append("--enable-%s" % ("shared" if self.options.shared else "static"))
+        args.append("--disable-%s" % ("static" if self.options.shared else "shared"))
+        self._autotools.configure(args=args)
+        return self._autotools
+
+    def build(self):
+        with tools.chdir(self._source_subfolder):
+            tools.save(path="VERSION", content="%s" % self.version)
+            autotools = self._configure_autotools()
+            autotools.make()
+
+    def package(self):
+        with tools.chdir(self._source_subfolder):
+            autotools = self._configure_autotools()
+            autotools.install()
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.remove_files_by_mask(self.package_folder, "*.la")
+
+    def package_info(self):
+        self.cpp_info.libs = ["sass"]
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            self.cpp_info.system_libs = ["dl"]
```

Comment:
```suggestion
            self.cpp_info.system_libs = ["dl", "m"]
```

libm is also linked to the shared library